data.json

```diff
@@ -25,6 +25,17 @@
       "url": "foobar.github.io/latest.zip",
       "resourcehub": "insert link to mod in resource hub here"
     },
+    "label-mod": {
+      "name": "Label Mod",
+      "name-internal": "label-mod",
+      "level": -1,
```

Comment:
-1 is for the Inapplicable rating on ResourceHub, I believe 0 would fit better here (green level)